docs/release-notes/2.44.0.md

```diff
@@ -0,0 +1,40 @@
+# Percona Monitoring and Management 2.44.0
+
+| **Release date** | December 9th, 2024                                                                                     |
+| -----------------| ----------------------------------------------------------------------------------------------- |
+| **Installation** | [Installing Percona Monitoring and Management](../quickstart/index.md) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+It enables you to observe the health of your database systems, explore new patterns in their behavior, troubleshoot them, and execute database management operations regardless of whether your databases are located on-premises or in the cloud.
+
+## Support for Percona Server for MongoDB 8.0  and Community MongoDB 8.0
```

Comment:
"Community MongoDB 8.0" 🤔  is this how we mention Mongo? Because, in theory, this, this will work with any Mongo (community or Enterprise). Maybe it's worth mentioning that it's explicitly to highlight that you can use PMM with the $(commercial) version of MongoDB too.

Reply:
We use these:
- MongoDB Community 
- MongoDB Enterprise Advanced 

In that case could be:
## Support for Percona Server for MongoDB, MongoDB Community, and MongoDB Enterprise 8.0